src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,103 @@
-package co.inventorsoft.academy.collections.model;
-
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
```

Comment:
Make all tests to pass, implement for arbitrary type of elements which implements Comparable